app/src/main/java/romina/valiunas/weatherapp/utils/WeatherReportAdapter.kt

```diff
@@ -6,7 +6,6 @@ import androidx.recyclerview.widget.RecyclerView
 import com.bumptech.glide.Glide
 import romina.valiunas.domain1.entities.Weather
 import romina.valiunas.weatherapp.databinding.ItemBinding
-import java.util.*
 
 class WeatherReportAdapter(private val briefWeatherReports: List<Weather>) :
     RecyclerView.Adapter<WeatherReportAdapter.ReportViewHolder>() {
```

Comment:
Lines 20-25 could be contained inside a "bind()" function inside ReportViewHolder

Reply:
function created